### DISCUSSION TITLE
not cyncing and not farming

### DISCUSSION THREAD

**Discussion by ashkanarvand** [Chia-Network/chia-blockchain#8440] [Support]
**I've been on my way to about 10T pilot  in two months. The problem I have right now is that not sync (Sometimes it's syncing) constantly and I can't get into farming.**
**Port 8444 connected** 
**### please help you solve this problem**

**Comment by loppefaaret** [comment]
check your `debug.log` file under you users folder `~\.chia\mainnet\log\` for any ERROR reports

**Reply by cypher619** [reply]
I am having the same issue.  I am seeing the following errors in my log
![chia errors](https://user-images.githubusercontent.com/75507117/133413205-55df8d64-4795-4b13-a010-8b7255d301b9.png)

**Reply by loppefaaret** [reply]
your case, your node is specifically saying it is failing the verify the weight proof given - if this persists, i can only advice to remove the database, and sync from 0... the blockchain database is found in your users folder `~\.chia\mainnet\db\` and is the file(s) starting with `blockchain*`

**Comment by loppefaaret** [comment]
are you plotting to the OS drive, the same drive that Chia is installed on ? it might saturate the drive, and that is causing a timeout during the initial weight proof needed before the sync starts

**Comment by BrandtH22** [comment]
@ashkanarvand , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
